src/platform/web/transport/transport-options.ts

```diff
@@ -31,4 +31,10 @@ export interface TransportOptions {
    * @defaultValue `true`
    */
   traceSip?: boolean;
+
+   /**
+   * can be used while creating websocket with array of protocols
```

Comment:
@kkbhanuchandar lets change the comment to be:
"Overriding the default protocols list for the WebSocket creation."

Reply:
done.
Also we are removing duplicates in protocols array to avoid disruption while connecting to WS.